assets/js/src/form_editor/store/actions.jsx

```diff
@@ -1,14 +1,14 @@
-export default {
-  toggleSidebar(toggleTo) {
-    return {
-      type: 'TOGGLE_SIDEBAR',
-      toggleTo,
-    };
-  },
-  changeFormName(name) {
-    return {
-      type: 'CHANGE_FORM_NAME',
-      name,
-    };
-  },
-};
+
+export function toggleSidebar(toggleTo) {
```

Comment:
For actions) I would keep then in one file or maybe later divide them to some groups. That is how they [are organized within Gutenberg repo](https://github.com/WordPress/gutenberg/blob/master/packages/edit-post/src/store/actions.js). Action function (in most cases) just returns a simple object. I find more convenient to have it in one place.

For reducers) I think we will eventually separate them and use [combineReducers](https://www.npmjs.com/package/@wordpress/data#combineReducers). I just thought
that for now, it is not necessary since we have just a few simple reducers.